### PR TITLE
Herm wip clean signin component

### PR DESCRIPTION
**Pull request by hermsicle**

### CODE REVIEW

File: src/containers/_RegistrationContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+	const onContinue = () => {
+		history.push('/signup/regform')
+	}
+
+	return (
+		<Registration>
+
+			<Registration.Inner>
+				<Registration.Image src="/images/registration.png"/>
```
Comment (awwmicky) [inline src/containers/_RegistrationContainer.jsx:17]: add an `alt` attribute for your images sire

File: src/containers/_RegistrationContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+					<Registration.Span>1</Registration.Span>
+					OF
+					<Registration.Span>3</Registration.Span>
+				</Registration.PreTitle>
+
+				<Registration.Title>Finish Setting up your account. </Registration.Title>
+				<Registration.SubTitle>
+					Netflix is personalized for you. Create a password to watch Netflix on any device at any time.
+				</Registration.SubTitle>
+				<Registration.Button
```
Comment (awwmicky) [inline src/containers/_RegistrationContainer.jsx:30]: do you prefer that format of Components: w/ text + eventHandler + other?
or in-line Component?

**Review by awwmicky — CHANGES_REQUESTED** [review]
Just a small clean up for you to do:
- in `src/components/Background`, add the '_' to you filenames
- in `src/containers/_SignupFormContainer`, change to jsx file extension
- in `src/pages/_*`, change to jsx file extension

let me know when you got the changes updated 👍

File: src/containers/_RegistrationContainer.jsx (hunk 2 above)
Comment (hermsicle) [reply src/containers/_RegistrationContainer.jsx:30]: Not sure what you mean,

File: src/containers/_RegistrationContainer.jsx (hunk 1 above)
Comment (hermsicle) [reply src/containers/_RegistrationContainer.jsx:17]: Just fixed it

**Pull request merged**